cluster/config-defaults.yaml

```diff
@@ -702,4 +702,11 @@ deployment_service_cf_auto_expand_enabled: "false"
 deployment_service_enabled: "true"
 
 # Standard storageclass gp3 volume type
-storageclass_standard_gp3: "true"
+storageclass_standard_gp3: "false"
```

Comment:
you probably didn't want to change this line?

Reply:
NO, don't change that :)

Reply:
oops, that happend during the merge :(